### ISSUE TITLE
leap input file?

### ISSUE THREAD

**Issue by hainm** [kmb413/DecoyDiscrimination#6]
can push leap input file you used? I saw [this](https://github.com/kmb413/DecoyDiscrimination/blob/master/decoys.set2.init/1aaj/ambercommands)  but this is not correct for our case.

**Comment by kmb413** [comment]
Yeah, check out DecoyDiscrimination/decoys.set2.init/1aaj/Rosetta_relaxed/NoH_5_empty_tag_979_0001_0001.tleap.in:

> source leaprc.ff14SBonlysc
> m = loadpdb NoH_5_empty_tag_979_0001_0001.pdb
> set default pbradii mbondi3
> saveamberparm m NoH_5_empty_tag_979_0001_0001.parm7 > NoH_5_empty_tag_979_0001_0001.rst7
> quit

Also, DecoyDiscrimination/scripts/ConvertToPDBandSubmitMinimizationJobs.py lines 110-115.

Is this correct?

**Comment by hainm** [comment]
yes. There are correct.

Can you hide/delete ambercommands to avoid confusion?

**Comment by kmb413** [comment]
Deleted! Sorry for the confusion.
